server/requirements.txt

```diff
@@ -41,4 +41,4 @@ wooper==0.4.1
 git+git://github.com/superdesk/eve@upstream-nk#egg=Eve==0.6-dev
 git+git://github.com/amagdas/tinys3@list-buckets#egg=tinys3
 -e hg+http://bitbucket.org/haypo/hachoir3/#egg=hachoir3
-git+git://github.com/liveblog/superdesk@V0.1#egg=superdesk
\ No newline at end of file
+git+git://github.com/liveblog/superdesk@V0.1#egg=Superdesk-Server
```

Comment:
Because this is the name of the superdesk python dependency
https://github.com/superdesk/superdesk/blob/master/setup.py#L20